modules/setting/setting.go

```diff
@@ -436,7 +436,7 @@ func forcePathSeparator(path string) {
 // This check is ignored under Windows since SSH remote login is not the main
 // method to login on Windows.
 func IsRunUserMatchCurrentUser(runUser string) (string, bool) {
-	if IsWindows {
+	if IsWindows || SSH.StartBuiltinServer {
```

Comment:
Seems to be a bug fix? Let's move it into a new PR.

Reply:
Done in #7215